frb_codegen/src/generator/dart/ty_sync_return.rs

```diff
@@ -0,0 +1,46 @@
+use crate::generator::dart::ty::*;
+use crate::ir::*;
+use crate::type_dart_generator_struct;
+use crate::utils::BlockIndex;
+
+type_dart_generator_struct!(TypeSyncReturnGenerator, IrTypeSyncReturn);
+
+impl TypeDartGeneratorTrait for TypeSyncReturnGenerator<'_> {
+    fn api2wire_body(&self, _: BlockIndex) -> Option<String> {
+        Some("/*unsupported*/".to_string())
+    }
+
+    fn wire2api_body(&self) -> String {
+        match self.ir {
+            IrTypeSyncReturn::Primitive(ref primitive) => match primitive {
+                IrTypePrimitive::Bool => "final dataView = ByteData.view(raw.buffer);
+                        return dataView.getUint8(0) != 0;"
```

Comment:
nit: Maybe we can create a simple helper function in `frb_dart`, and call that function from here. Then, we avoid generating duplicated code.

Reply:
Got it!